### PR TITLE
Refactoring: Added the config.yaml file, moved the COCO CLASSES out o…

### PR DESCRIPTION
**Pull request by Lemeri02**
Refactoring: Added the config.yaml file, moved the COCO CLASSES out of the code and into the yaml file, etc

### CODE REVIEW

**Review by vadim328 — APPROVED** [review]
Good!

File: main.py [hunk 1]
```diff
@@ -82,5 +74,5 @@ def detect_objects(image_data, name):
     result = creating_frame(prepared_data, image)
     filename = f'{uuid.uuid4()}_{name}'
     image.save(f'./static/{filename}')
-    result.append({'image_url': f'{config["BASE_URL"]}/static/{filename}'})
+    result.append({'image_url': f'{env["BASE_URL"]}/static/{filename}'})
```
Comment (AkininD) [inline main.py:77]: Good!

File: .env [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# This .env file should be in .gitignore and not accessible to the public
```
Comment (AkininD) [inline .env:1]: Good!

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Example file
```
Comment (AkininD) [inline .env.example:1]: Good!

File: config.yaml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  - chair
+  - couch
+  - potted plant
+  - bed
+  - N/A
+  - dining table
+  - N/A
+  - N/A
+  - toilet
+  - N/A
```
Comment (AkininD) [inline config.yaml:75]: Good!

File: helpers.py [hunk 5]
```diff
@@ -27,11 +27,8 @@ def detect(im, model, transform):
     # if you want to use images with an aspect ratio outside this range
     # rescale your image so that the maximum size
     # is at most 1333 for best results
-    assert img.shape[-2] <= 1600 \
-           and img.shape[-1] <= 1600, 'demo model only supports images' \
-                                      'up to 1600 pixels on ' \
-                                      'each side'
-
+    error_message = 'demo model only supports images up to 1600 pixels on each side'
```
Comment (AkininD) [inline helpers.py:30]: excellent!

File: helpers.py [hunk 6]
```diff
@@ -27,11 +27,8 @@ def detect(im, model, transform):
...
+    assert img.shape[-2] <= 1600 and img.shape[-1] <= 1600, error_message
```
Comment (AkininD) [inline helpers.py:31]: excellent!

File: requirements.txt [hunk 7]
```diff
@@ -6,4 +6,4 @@ pydantic
 python-multipart
 python-dotenv
 pytest
-httpx
+httpx
```
Comment (AkininD) [inline requirements.txt:9]: excellent!

**Review by AkininD — APPROVED** [review]

**Pull request merged**